src/controllers/objecting.entity.name.controller.ts

```diff
@@ -57,7 +57,12 @@ const showErrorsOnScreen = (errors: Result, res: Response) => {
       }
       errorListData.push(govUkErrorData);
     });
+
+  const fullNameValue: string = req.body.fullName
   return res.render(Templates.OBJECTING_ENTITY_NAME, {
+    fullNameValue,
```

Comment:
can we add a unit test to cover this new code?

Reply:
Unit test coverage added.

Reply:
Will need a unit test to cover the retention of the name as well

Reply:
Scratch that, it's covered